### ISSUE TITLE
Consolidate Asciidoclet documentation

### ISSUE THREAD

**Issue by msgilligan** [asciidoctor/asciidoclet#87]
There is a high-degree of duplication in the Asciidoclet documentation spread over three places:

* The Github project [README.adoc](https://github.com/asciidoctor/asciidoclet/blob/master/README.adoc)
* The [1.5.0 Release Notes](https://asciidoctor.org/news/2014/09/09/asciidoclet-1.5.0-released/)
* The [Asciidoclet Guide](https://asciidoctor.org/docs/asciidoclet/) on Asciidoctor.org

The source code for these docs should be consolidated for ease-of-maintenance and it should probably be published to a single place as well.

My recommendation is that the consolidated source be stored in `src/docs/asciidoc/asciidoclet-guide.adoc` within this project. That way it can more easily be kept in sync with the latest version of the code.

On the publication/distribution side, I'm looking for guidance from @mojavelinux and @johncarl81 as to what they think is best. I like the idea of having a nicely rendered and styled **Asciidoclet Guide** that can be linked to from the README, release notes, etc. I don't understand the publication/aggregation process for Asciidoctor.org and if it can pull content from this repository. Are there plans to use Antora going forward?

**Comment by msgilligan** [comment]
@mojavelinux @johncarl81 Any thoughts on how we should approach this? Where should the single/consolidated document source live? Where should it published?

**Comment by johncarl81** [comment]
The 1.5.0 Release Notes (and [Write Javadocs in AsciiDoc with Asciidoclet](https://asciidoctor.org/news/2013/06/03/asciidoclet-announcement/)) are blog entries and shouldn't be considered true documentation.

I like having the main project documentation, especially for a small project like this one, as the README for the project.  This makes it easy to find because it's rendered as the home page of the project on github.  The Asciidoclet Guide is simply a copy of the README and exists because Asciidoclet is part of the Asciidoctor OS community.  If we need to bolster or consolidate the documentation I'd start with the README and add additional sources as links if necessary.

**Comment by mojavelinux** [comment]
You can find the repository where this is all happening here:

https://github.com/asciidoctor/docs.asciidoctor.org

You're welcome to join the party at any time. Once you have an Antora docs structure set up somewhere, perhaps in a branch, we can point Antora at it (antora-playbook.yml) and those docs will be included in the site.

**Comment by msgilligan** [comment]
Sorry I've been AWOL lately. (I've been very busy with work and family. I'm looking forward to having more time to help out, but it might be another month or so.) I'm looking forward to playing with Antora.

**Comment by mojavelinux** [comment]
Yeah! It's a good time to get back into it because things are really starting to get more organized.

The new docs site will be available later this month and should serve as an excellent reference. Antora is designed to be very simple (once the pipeline is set up), so it shouldn't require too much effort to chop up the docs into individual pages and get them published. And I'll be here to help out!

**Comment by apj68** [comment]
@msgilligan , I can take a stab at consolidating the docs.  I'm not much an expert but I love the whole platform and I'm somewhat familiar with Antora (read: basic user).  I've just ran around the horn trying to get `asciidoclet` working with `asciidoctorj-diagram` which leads to a fantastic goose chase over the web since virtually every sample out there are instructing for pre- or early-asciidoctorj and many without `diagram` so it can be confusing.  When I finally got the configuration right it was giving jruby errors which resolved by using @devlauer's pull request (#104) and updated asciidoctorj libraries).  To that I needed to add 'fr.jmini.asciidoctorj:file-logger:1.0.0' and 'com.google.guava:guava:20.0' to the classpath.  It would probably help others in the future if you could merge that.

**Comment by mojavelinux** [comment]
> I can take a stab at consolidating the docs.

:tada: 

If you need help with Antora, don't hesitate to @ me.

**Comment by apj68** [comment]
It looks like I'm too late to help with this one.. 😄   Asciidoclet looks to have already been migrated to the [main Antora site](https://docs.asciidoctor.org/asciidoclet/latest/) by @abelsromero.  Per [Dan's comment here](https://github.com/asciidoctor/asciidoclet/issues/87#issuecomment-429207438) I could possibly prune down the github README.adoc and make reference to the main docs but @abelsromero seems to already have done a great job and in all honesty I think the gist of this issue is already complete.

**Comment by mojavelinux** [comment]
You're absolutely right. @abelsromero did a fantastic job of migrating the docs for inclusion in the new site. I do think the README could be further simplified so that there's no duplication, but you may want to consider making that a separate issue.

**Comment by abelsromero** [comment]
> I could possibly prune down the github README.adoc and make reference to the main docs

Absolutely, go for it! I just stepped in for the bare minimum to keep the train going.
